packages/bitcore-client/src/wallet.ts

```diff
@@ -413,7 +430,10 @@ export class Wallet {
       gasLimit: params.gasLimit || 200000,
       data: params.data,
       tokenAddress: tokenContractAddress,
-      contractAddress: params.contractAddress
+      contractAddress: params.contractAddress,
+      replaceByFee: params.replaceByFee,
```

Comment:
This is nice to add but it appears CWC isn't ready to handle RBF. Fine to just keep it in preparation for adding full support.

Reply:
I changed CWC to support this locally but forgot to commit. Pushed now.